libc/utils/docgen/signal.json

```diff
@@ -0,0 +1,21 @@
+{
+  "macros": [
+    "__STDC_VERSION_FENV_H__"
```

Comment:
7.14 also mentions a few addition macros:
- `SIG_DFL`
- `SIG_ERR`
- `SIG_IGN`

and a few more.  I actually find section `B.13 Signal handling <signal.h>` easier to view a concise summary of what's expected to be found where (though in n3096 it lacks a mention of `__STDC_VERSION_FENV_H__` which seems like a minor mistake in the spec).

Please add those, too, even if we aren't generating documentation for them yet today.

Reply:
Ah, got it, I'll fix it.

Reply:
done for that.